TheWitcher.Tests/HeroServiceTests.cs

```diff
@@ -0,0 +1,172 @@
+﻿using DataAccessLayer.Interfaces;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using TheWitcher.Business;
+using TheWitcher.Core;
+using TheWitcher.DataAccess.Interfaces;
+using TheWitcher.Domain.Mappers;
+using TheWitcher.Domain.Models;
+using Xunit;
+
+namespace TheWitcher.Tests
+{
+    public class HeroServiceTests
+    {
+        [Fact]
+        public void GetHeroDTOTestByNegativeId()
+        {
+            var mockUnitOfWork = new Mock<IUnitOfWork>();
+            var mockIClothesRepository = new Mock<IRepository<Clothes>>();
+            mockIClothesRepository.Setup(x => x.GetItem(4)).Returns(new Clothes());
+            mockUnitOfWork.Setup(x => x.Clothes).Returns(mockIClothesRepository.Object);
+            var mockMapHeroes = new Mock<IMapHeroes>();
+            var heroService = new HeroService(mockUnitOfWork.Object, mockMapHeroes.Object);
+            heroService.BuyClothes(4, 4);
+
+            var resultOfTest = heroService.GetHeroDTO(-1);
+
+            Assert.Null(resultOfTest);
+
+        }
+
+        [Fact]
+        public void GetHeroDTOTestByPositiveId()
+        {
+            var mockUnitOfWork = new Mock<IUnitOfWork>();
+            var mockMapHeroes = new Mock<IMapHeroes>();
+            var mockIHeroRepository = new Mock<IRepository<Heroes>>();
+                        
+            Heroes hero = new Heroes()
+            {
+                HeroMoney = 50,
+                AvailableWeight = 30,
+                HeroName = "Geralt",
+                HeroDescription = "Ведьмак школы волка",
+                HeroLevel = 1,
+                Playable = true
+            };
+            mockIHeroRepository.Setup(x => x.GetItem(4)).Returns(hero);
+            mockUnitOfWork.Setup(x => x.Hero).Returns(mockIHeroRepository.Object);
+            mockMapHeroes.Setup(x => x.AutoMapHeroes(hero)).Returns(new HeroesDTO());
+            var heroService = new HeroService(mockUnitOfWork.Object, mockMapHeroes.Object);
+
+
+            var resultOfTest = heroService.GetHeroDTO(4);
+
+
+            HeroesDTO heroesDTO = new HeroesDTO();
+            Assert.IsType(heroesDTO.GetType(), resultOfTest);
```

Comment:
1. Use typeof() instead of GetType();
2. Check that values are correct, not only type check